internal/pkg/cli/deploy/workload.go

```diff
@@ -249,14 +256,16 @@ func newWorkloadDeployer(in *WorkloadDeployerInput) (*workloadDeployer, error) {
 	}
 
 	cfn := cloudformation.New(envSession, cloudformation.WithProgressTracker(os.Stderr))
+
 	return &workloadDeployer{
 		name:                     in.Name,
 		app:                      in.App,
 		env:                      in.Env,
 		image:                    in.Image,
 		resources:                resources,
 		workspacePath:            ws.Path(),
-		fs:                       afero.NewOsFs(),
+		uri:                      uri,
+		fs:                       &afero.Afero{Fs: afero.NewOsFs()},
```

Comment:
+1 to parag's question. Is it to make sure that the fs dependency is a pointer and not an object?

Reply:
I wonder this is done by editor automatically lol. I will revert this `fs`field.